react/ActionMenu/index.jsx

```diff
@@ -125,13 +129,11 @@ class ActionMenu extends Component {
   }
 
   handleMenuRef = menu => {
-    // FIXME
-    this.menuNode = ReactDOM.findDOMNode(menu) // eslint-disable-line react/no-find-dom-node
+    this.menuNode = menu
   }
 
   handleWrapperRef = wrapper => {
-    // FIXME
-    this.wrapperNode = ReactDOM.findDOMNode(wrapper) // eslint-disable-line react/no-find-dom-node
+    this.wrapperNode = wrapper
```

Comment:
I think we can now put them directly in the render code

Reply:
I prefer not to create functions in the render method. It's not based on scientific benchmarks.

Reply:
Ok